content/overview/pricing/free-limited-sandbox.md

```diff
@@ -8,44 +8,42 @@ aliases:
   - /overview/pricing/free-limited-sandbox
 ---
 
-Anyone with a U.S. federal government email address (.gov, .mil, or .fed.us) can evaluate cloud.gov for free using a sandbox space.
-
-A sandbox is a free, limited way to try cloud.gov and see if it might suit your team's needs. The sandbox gives you a sense of how cloud.gov would work for the applications your team builds. From the [setup process]({{< relref "docs/getting-started/setup.md" >}}) through [deploying an app]({{< relref "docs/getting-started/your-first-deploy.md" >}}), it works similarly to using a [paid access package]({{< relref "overview/pricing/rates.md" >}}), with [some limitations](#sandbox-limitations).
+A sandbox is a free account that you can use to see if cloud.gov might suit your team’s needs. From the [setup process]({{< relref "docs/getting-started/setup.md" >}}) through [deploying an app]({{< relref "docs/getting-started/your-first-deploy.md" >}}), it works similarly to using a [paid access package]({{< relref "overview/pricing/rates.md" >}}), with [some limitations](#sandbox-limitations).
```

Comment:
Ah yeah, I meant my comment with emphasis on _space_ rather than _free_. I agree that "free" is already well-communicated in general and could even be repeated less. The problem is that calling a sandbox a free _account_ is confusing. The limitations on sandbox spaces apply whether or not your account also has access to paid orgs.

Specifically I mean that the change from "A sandbox is a free, limited way" to "A sandbox is a free account" adds some confusion because a sandbox is not an "account", it's a "space".